ArduPlane/radio.cpp

```diff
@@ -116,60 +116,6 @@ void Plane::init_rc_out_aux()
     SRV_Channels::setup_failsafe_trim_all_non_motors();
 }
 
-/*
-  check for pilot input on rudder stick for arming/disarming
-*/
-void Plane::rudder_arm_disarm_check()
-{
-    const int16_t rudder_in = channel_rudder->get_control_in();
-    if (rudder_in == 0) {
-        // remember if we've seen neutral rudder, used for VTOL auto-takeoff
-        seen_neutral_rudder = true;
-    }
-	if (!arming.is_armed()) {
-		// when not armed, full right rudder starts arming counter
-        if (rudder_in > 4000) {
-			uint32_t now = millis();
-
-			if (rudder_arm_timer == 0 ||
-				now - rudder_arm_timer < 3000) {
-
-				if (rudder_arm_timer == 0) {
-                    rudder_arm_timer = now;
-                }
-			} else {
-				//time to arm!
-				arming.arm(AP_Arming::Method::RUDDER);
-                rudder_arm_timer = 0;
-                seen_neutral_rudder = false;
-                takeoff_state.rudder_takeoff_warn_ms = now;
```

Comment:
This is lost I think. May not be a issue, just means we get the first "Takeoff waiting for rudder release" immediately instead of after 3 seconds.

Reply:
Fixed